spec/features/notes_on_merge_requests_spec.rb

```diff
@@ -50,18 +51,22 @@
       end
 
       it 'should be added and form reset' do
-        should have_content("This is awsome!")
+        is_expected.to have_content("This is awsome!")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.